.circleci/config.yml

```diff
@@ -180,6 +180,10 @@ jobs:
         name: prepare sql test database
         command: psql -h 127.0.0.1 -p 5432 -U postgres < .docker/initdb.sql
 
+    - run:
+        name: warm up the cache
+        command: pip3 install molgenis-emx2-pyclient numpy==1.23.4
```

Comment:
Why `numpy==1.23.4`? The current version is at [2.1.3](https://pypi.org/project/numpy/)